workers/price-reporter/src/worker.rs

```diff
@@ -93,6 +93,7 @@ impl Worker for PriceReporter {
 
     fn start(&mut self) -> Result<(), Self::Error> {
         // Spawn a tokio thread pool to run the manager in
+        // NOTE(@akirillo): Do we need this?
```

Comment:
@joeykraut out of scope for this PR, but it appears that we construct the Tokio runtime below and never use it. I'm happy to remove this if I'm not missing anything